src/TLuaInterpreter.cpp

```diff
@@ -13274,27 +13308,47 @@ void TLuaInterpreter::loadGlobal()
     {
         // For the installer we do not go down a level to search for this. So
         // we check again for the user case of a windows install.
+#if QT_VERSION >= 0x050200
+        // overload previous behaviour to check by absolute path as well
+        // TODO this sould be cleaned up and refactored to just use an array and a for loop
+        path = QCoreApplication::applicationDirPath() + "/mudlet-lua/lua/LuaGlobal.lua";
+        if ( ! QFileInfo::exists(path) ) {
+            path = "mudlet-lua/lua/LuaGlobal.lua";
+        }
+#else
         path = "mudlet-lua/lua/LuaGlobal.lua";
+#endif
         error = luaL_dofile( pGlobalLua, path.toLatin1().data() );
```

Comment:
We might want to substitute `path.toUtf8().constData()` for `path.toLatin1().data()` - (use `utf8()` for those Windows machines who's user name and thus path contains non-ASCII characters; and `constData()` because we do not need to/want to/can change the supplied string which is the raw characters of `path` IIRC).